src/main/java/com/shrek/task00002/Runner.java

```diff
@@ -0,0 +1,22 @@
+package com.shrek.task00002;
+
+public class Runner implements Comparable<Runner> {
+
+    private String name = "";
+    private int time = 0;
+    public Runner(String name, int time){
+        this.name = name;
+        this.time = time;
+    }
+    public String getName(){
+        return name;
+    }
+
+    public int getTime() {
+        return time;
+    }
+
+    public int compareTo(Runner r) {
```

Comment:
Use `@Override` annotation. It helps to prevent programmer mistakes with overriding